devlib/module/cpufreq.py

```diff
@@ -414,10 +414,12 @@ def trace_frequencies(self):
     def get_domain_cpus(self, cpu):
         """
         Get the CPUs that share a frequency domain with the given CPU
+
+        Returned as a tuple of CPU IDs
         """
         if isinstance(cpu, int):
             cpu = 'cpu{}'.format(cpu)
 
         sysfile = '/sys/devices/system/cpu/{}/cpufreq/affected_cpus'.format(cpu)
 
-        return [int(c) for c in self.target.read_value(sysfile).split()]
+        return tuple(int(c) for c in self.target.read_value(sysfile).split())
```

Comment:
I don't think this should return a tuple. A tuple implies a fixed size and a fixed meaning for the value at each index. This is not the case here.